docs/automated-testing/getting-started.md

```diff
@@ -0,0 +1,125 @@
+
+## Installing the package
+
+Our package is available at [https://github.com/Regression-Games/RegressionGames.Unity.Testing](https://github.com/Regression-Games/RegressionGames.Unity.Testing).
+Import it into your game by clicking **Window > Package Manager** in the Unity menu,
+and then clicking the **+** button in the top left corner of the Package Manager window.
+Select **Add package from git URL** and paste in the following URL:
+
+```
+https://github.com/Regression-Games/RegressionGames.Unity.Testing.git?path=src/gg.regression.unity.testing#v0.0.1
+```
+
+* **NOTE**: The package uses [TextMeshPro](https://docs.unity3d.com/Manual/com.unity.textmeshpro.html) to provide UI elements. If you are prompted by Unity to add TextMeshPro assets to your project, please add them.
+* **NOTE**: The package uses [Newtonsoft Json](https://docs.unity3d.com/Packages/com.unity.nuget.newtonsoft-json@3.1/manual/index.html) to serialize state for recordings.
+
+![The Unity Package Manager dialog with the "Add package from git URL" option selected.](img/getting-started/add-package-from-git.png)
+
+## Adding the Automation Controller and components to your scene
+
+Once the package is installed, you need to add an Automation Controller to the scene you wish to automate.
+The Automation Controller is the hub for all automation in a scene.
+Bots are spawned as children of the Automation Controller,
+and Entity Discoverers (see below) should be added as children of the Automation Controller.
+The Controller also provides APIs for spawning bots and managing recordings (via a separate, Automation Recorder component attached to the same object).
+
+To add the Automation Controller to your scene, click **GameObject > Regression Games > Automation Controller** in the Unity menu.
+
+![The "Automation Controller" option in the Regression Games submenu of the GameObject menu](img/getting-started/add-automation-controller-menu.png)
+
+**By default**, the Automation Controller and all it's children will mark themselves as [DontDestroyOnLoad](https://docs.unity3d.com/ScriptReference/Object.DontDestroyOnLoad.html). This ensures that bots continue to run when you load a new scene in your game. You can disable this behavior by unchecking the "Dont Destroy On Load" checkbox on the Automation Controller component.
+
+![The Automation Controller component with the "Dont Destroy On Load" checkbox checked](img/getting-started/ac-dont-destroy-on-load.png)
+
+## Adding Entity Discoverers to your scene
+
+In order to automate entities in your game, the Automation Controller needs to be able to find them.
+The Automation Controller uses Entity Discoverers to find entities in the scene that can be automated.
+The package provides a few built-in Entity Discoverers, but you can also create your own.
+To add one of the built-in discoverers, open the **GameObject > Regression Games > Discovery** menu and select the discoverer you want to add:
```

Comment:
> Here, also be sure to add that you should do this on the automation controller, unless it can be done at the top level as well (but we should at least tell them somewhere to click to get that menu)

If you click that menu item without a node selected, it automatically targets the automation controller. Otherwise, it should give you a fairly clear error telling you the problem.

> I see... this comes after the picture. I'd put it before the picture so that we avoid the scenario I just had where I didn't scroll further to see more and got stuck on that

I didn't want to put it the image above since the text that describes the image should generally precede it. I can do that if you think it's clearer though.